tedge/src/cli/certificate/error.rs

```diff
@@ -100,3 +105,71 @@ impl CertError {
         }
     }
 }
+
+// Our source of error here is quite deep into the dependencies and we need to dig through that to get to our certificates validator errors which are Box<&dyn Error> through 3-4 levels
```

Comment:
Do you think it's worth raising an issue on reqwest to allow users to get this kind of information in a stable way?

Reply:
In earlier versions of `reqwest` the internal dependencies were reexported but they now have hidden all of those, maybe due to `rustls` allows to use different backends to do certificates verification.
There is quite old feature request which asks for similar, but there is no activity on it since Jan 2020.
https://github.com/seanmonstar/reqwest/issues/757